src/systems/apply_joint_force/ApplyJointForce.cc

```diff
@@ -156,23 +156,14 @@ void ApplyJointForce::PreUpdate(const UpdateInfo &_info,
 
   // Update joint force
   //! [jointForceComponent]
-  auto force = _ecm.Component<components::JointForceCmd>(
-      this->dataPtr->jointEntity);
+  auto force = _ecm.ComponentDefault<components::JointForceCmd>(
```

Comment:
I'd rather not make this kind of change in a code freeze. We can merge this after Ionic is released